scripts/update-search-index.sh

```diff
@@ -74,6 +81,13 @@ function index_docs() {
   #Verify parameters
   assert_not_empty "--api-key" "$api_key"
 
+  echo "Updating search index with docs-sourcer using prefix: $index_prefix"
+  # Update Algolia index using docs-sourcer. This is the new method of indexing.
+  # In order to add plugins to this list just space separate them.
+  ALGOLIA_APP_ID="$ALGOLIA_APP_ID" ALGOLIA_API_KEY="$api_key" ALGOLIA_INDEX_PREFIX="$index_prefix" yarn regenerate --plugins docs-indexer
```

Comment:
Do we need to clone the docs-sourcer somewhere in the job first for this to work?

Reply:
Nope - the plugin hits GitHub via API. So we should be decent

Reply:
:facepalm: sorry I saw the dependency then immediately decided i didn't see it, apparently